### PR TITLE
Add Github workflow, upgrade Elixir, and remove /ops/env route

### PR DESCRIPTION
**Pull request by Stratus3D**
[Card](https://rentpath.atlassian.net/browse/SRV-4883)

* Add Github CI workflow
* Remove unused and unimplemented `/ops/env` route
* Upgrade Erlang and Elixir
* Address compiler warnings
* Add formatter configuration and run formatter
* Upgrade to latest version of credo and update credo config

### CODE REVIEW

File: lib/build_details.ex [hunk 1]
```diff
@@ -31,28 +31,30 @@ defmodule ExOps.BuildDetails do
...
   defp resolve_path(info_file) do
     file_path = Map.get(info_file, :path)
+
     case Map.get(info_file, :type) do
       :priv_dir ->
         application = Map.get(info_file, :application)
         :erlang.iolist_to_binary([:code.priv_dir(application), "/", file_path])
-      :path  ->
+
+      :path ->
```
Comment (Stratus3D) [inline lib/build_details.ex:40]: All of the changes to Elixir source code were made by the formatter.

**Review by Ebtoulson — APPROVED** [review]

File: .credo.exs [hunk 2]
```diff
@@ -71,57 +71,77 @@
...
         {Credo.Check.Readability.AliasOrder},
         {Credo.Check.Readability.FunctionNames},
         {Credo.Check.Readability.LargeNumbers},
         {Credo.Check.Readability.MaxLineLength, max_length: 120},
         {Credo.Check.Readability.ModuleAttributeNames},
         {Credo.Check.Readability.ModuleDoc},
         {Credo.Check.Readability.ModuleNames},
+        # Unfortunately there is a bug in the NestedFunctionCalls check I wrote
+        # so it must be disabled for now. Enable it next year after the fix has
+        # been merged.
```
[the remark this answers is not in the document]
Comment (Stratus3D) [reply .credo.exs]: PR updated.

File: mix.exs [hunk 3]
```diff
@@ -5,9 +5,9 @@ defmodule ExOps.Mixfile do
     [
       app: :ex_ops,
       version: "2.2.0",
-      elixir: "~> 1.10",
-      build_embedded: Mix.env == :prod,
-      start_permanent: Mix.env == :prod,
+      elixir: "~> 1.14",
+      build_embedded: Mix.env() == :prod,
```
Comment (KevinMarth) [inline mix.exs:9]: I don't think `build_embedded` is needed now. https://github.com/elixir-lang/elixir/issues/11715

**Review by pasha — APPROVED** [review]

File: mix.exs (hunk 3 above)
Comment (Stratus3D) [reply mix.exs]: PR updated.

**Review by KevinMarth — APPROVED** [review]

**Pull request merged**